manifests/2.19.0/opensearch-2.19.0-test.yml

```diff
@@ -69,6 +69,7 @@ components:
       additional-cluster-configs:
         path.repo:
           - /tmp
+          - /testSnapshotFolder
```

Comment:
Isn't this suppose to be for k-NN component?
Should be under that. Also `/tmp` default exists on most of the platforms. Is `testSnapshotFolder` being created by some gradle action?
Adding @peterzhuamazon @rishabh6788

Reply:
I didnt realize they add it under index-management.